docs/library/tables-and-data-grids/ag_grid.md

```diff
@@ -561,3 +561,144 @@ The following props are available for `column_defs` as well as many others that
 - `checkbox_selection`: `bool | None`: Set to true to render a checkbox for row selection.
 - `cell_editor`: `AGEditors | str | None`: Provide your own cell editor component for this column's cells. (Check out the Editing section of this page for more information)
 - `cell_editor_params`: `dict[str, list[Any]] | None`: Params to be passed to the cellEditor component.
+
+
+
+## Functionality you need is not available in Reflex
+
+Since Reflex AG Grid is wrapping the underlying AG Grid library, there is much more functionality available that is currently not exposed in Reflex. Check out this [documentation](https://www.ag-grid.com/react-data-grid/reference/) for more information on what is available in AG Grid.
+
+As Reflex does not expose all the functionality of AG Grid, you can use `ag_grid.api()`, which is a property of the `ag_grid` component, to access the underlying AG Grid API. This allows you to access the full functionality of AG Grid.
```

Comment:
> which is a property of the `ag_grid` component

Technically this is also true, but this page describes the `api` class that's hanging off the `ag_grid` _namespace_

Reply:
fixed